ReadMe.md

```diff
@@ -0,0 +1,99 @@
+# LibWalter v0.1
+
+LibWalter is a code collection which aims to fill in the gaps in the BeOS API, released under the MIT license. Designed to be compatible with BeOS R5, Zeta 1.x, and Haiku.
+
+### 0. Index:
```

Comment:
I am pretty sure, I made the comment before, but cannot find it anymore...:

The headings should be one level up, e.g. `### 0. Index` becomes `## 0. Index`